docs/extending_filer.rst

```diff
@@ -308,3 +308,29 @@ Set ``FILER_IMAGE_MODEL`` to the path of your custom model:
 .. code-block:: python
 
     FILER_IMAGE_MODEL = 'myapp.CustomImage'
+
```

Comment:
Please add these sections:
`Overview` - a general overview of this functionality and it's architecture
`Why would I want to use this?` - with some user journey examples